packages/manfred2html/src/engine/template-a/html-parts/about-me-section/about-me-section.part.ts

```diff
@@ -0,0 +1,20 @@
+import ejs from 'ejs';
+import { ProfileSectionVm, mapFromMacCvToProfileSectionVm } from '@lemoncode/manfred-common/profile-section';
+import { ManfredAwesomicCV, Settings, Language } from '@/model';
+import { englishAboutMeLabels, spanishAboutMeLabels } from './labels';
+import aboutMe from './about-me-section.ejs?raw';
+
+export const generateAboutMeSection = (cv: ManfredAwesomicCV, settings: Settings): string => {
+  const profileSectionVm = mapFromMacCvToProfileSectionVm(cv);
+
+  return generateAboutMeSectionInner(profileSectionVm, settings.language);
+};
+
+const generateAboutMeSectionInner = (profileSectionVm: ProfileSectionVm, language: Language = 'es'): string => {
```

Comment:
Los 'es' están repetidos por varios sitios, create una constante

const __spanishLanguage = 'es';

Y usala en todos sitios

Reply:
create un fichero más arriba que se llame

engine.const.ts

Y lo usas en todos sitios

Reply:
is fixed